build.gradle.kts

```diff
@@ -27,8 +28,8 @@ subprojects {
         jcenter()
         mavenLocal()
         maven { url = uri("https://plugins.gradle.org/m2/") }
-        maven { url = uri("http://repo1.maven.org/maven2") }
+        maven { url = uri("https://repo1.maven.org/maven2") }
         maven { url = uri("https://oss.sonatype.org/content/groups/staging/") }
```

Comment:
Do we need staging repo when using Gradle Composite Builds?

Reply:
It might be even removed. I fixed the protocol here, because since 2020 maven repo without `https` returns error code without redirect.